### DISCUSSION TITLE
How to use jumbotron?

### DISCUSSION THREAD

**Discussion by nousernameavailableanymore** [mkdocs/mkdocs#2485] [Q&A]
The index.md of mkdocs.org seems to use jumbotron "mode", however, I was not able to find any documentation on how to use that.
Can anyone point me into the right direction or provide some explanation?
Thank you.

**Comment by oprypin** [comment]
It is just custom HTML being put directly into the source of the site.
https://github.com/mkdocs/mkdocs/blame/master/docs/index.md#L21
There is no real support for it, just that the "mkdocs" theme is based on Bootstrap, and so you can include whatever HTML makes sense for Bootstrap, and it will work.

**Reply by oprypin** [reply]
Another example of custom HTML, this time for mkdocs-material theme:

https://crystal-lang.org/reference/
https://github.com/crystal-lang/crystal-book/blame/b65c7ba74d8776b08f694479ba8c77cc38eca466/docs/README.md#L16
https://github.com/crystal-lang/crystal-book/blob/b65c7ba74d8776b08f694479ba8c77cc38eca466/docs/assets/style.css#L31
